src/main/kotlin/systems/ajax/englishstudytelegrambot/dto/Response.kt

```diff
@@ -0,0 +1,46 @@
+package systems.ajax.englishstudytelegrambot.dto
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+interface GettingPartOfAdditionalInfoAboutWord {
+
+    fun getPartOfAdditionalInfoAboutWord(): String
```

Comment:
In kotlin we can declare abstract property
```suggestion
    val partOfAdditionalInfoAboutWord: String
```

In classes: 
```kotlin
data class AudioForWordResponse(
    @JsonProperty("fileUrl") val fileUrl: String
) : GettingPartOfAdditionalInfoAboutWord {

    override val partOfAdditionalInfoAboutWord: String = fileUrl
}
```